clients/web/static/lib/bootstrap/fonts/glyphicons-halflings-regular.svg

```diff
@@ -0,0 +1,229 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
I'm going to experiment to see how hard it is to override the CSS of the datepicker widget to use our existing fontello icons so we can get rid of these.